src/java.base/unix/native/libjava/ProcessImpl_md.c

```diff
@@ -57,7 +57,7 @@
  *   changing paths...
  * - then exec(2) the target binary
  *
- * There are three ways to fork off:
+ * On the OS-side are four ways to fork off:
```

Comment:
I would mention the two ways that are implemented, only mentioning vfork after the supported mechanisms to inhibit someone wanting to go backwards.